backstop/tests/side-navigation.html

```diff
@@ -4,51 +4,58 @@
 -->
 <html id="theme">
   <head>
-    <title>Side Navigation</title>
+    <title>Side Navigation | iTwinUI</title>
+    <link
+      rel="icon"
+      href="../assets/favicon.ico"
+    />
     <link
       rel="stylesheet"
       href="../../lib/css/global.css"
     />
     <link
       rel="stylesheet"
-      href="../../lib/css/side-navigation.css"
+      href="../../lib/css/button.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/tooltip.css"
     />
     <link
       rel="stylesheet"
-      href="../../lib/css/tooltips.css"
+      href="../../lib/css/side-navigation.css"
     />
     <link
       rel="stylesheet"
       href="../assets/demo.css"
     />
     <style>
-    #demo-sidenav-container.iui-expanded
-    .iui-sidenav-tooltip-container {
-      display: none;
-    }
-    /* tooltip */
-    .iui-sidenav-item-container .iui-sidenav-tooltip-container {
-      margin-left: 64px;
-      position: absolute;
-      opacity: 0;
-      display: none;
-    }
-
-    .iui-active > .iui-sidenav-tooltip-container {
-      margin-left: 62px;
-    }
-
-    .iui-sidenav-item-container:hover .iui-sidenav-tooltip-container {
-      opacity: 1;
-      display: block;
-    }
-
-    .iui-disabled:hover > .iui-sidenav-tooltip-container {
-      opacity: 0;
-      display: none;
-    }
-    </style>
+      section {
+        height: 666px;
+      }
+
+      body {
+        background-color: var(--iui-color-background-2) !important;
```

Comment:
I don't remember but what was the problem that it needs to be set to this color?

Reply:
[Here's the original discussion.](https://github.com/iTwin/iTwinUI/pull/33#discussion_r625873527)  Basically the header and side nav will most often be placed above background-2.  Most other components can be placed above background-1 or 2.